apps/web/pages/results.js

```diff
@@ -14,10 +15,16 @@ type Props = {
 };
 
 const results = ({ router }: Props) => {
+  function onSubmit(query) {
```

Comment:
This naming was a little bit confusing for me, for the first time, now I understand.
But I am thinking if here want to be better call that something like `changeUrl` or something like and passes it as an on submit into the: 
```javascript 
<SeachForm onSubmit={this.props.changeUrl} />
``` 

For my, it will be more readable. It is just Idea what do you think.

Reply:
Totally agree, naming is a bit confusing here

Reply:
done